laser_encoders/__init__.py

```diff
@@ -0,0 +1 @@
+from laser_encoders.download_models import initialize_encoder, initialize_tokenizer
```

Comment:
We'll need to add the copyright header to this file and to `language_list.py`.

Reply:
Alright